src/screens/Flashcards/Flashcards.js

```diff
@@ -0,0 +1,72 @@
+import React, { useEffect, useState } from "react";
+import styles from "./Flashcards.module.css";
+
+import { FlashcardsIntro, Flashcard} from 'components';
+import { Questions } from './QuestionsBase.js';
+
+export function Flashcards() {
+  const [shuffledQuestions, setShuffledQuestions] = useState(Questions);
+  const [showIntro, setShowIntro] = useState(true);
+  const [known, setKnown] = useState(JSON.parse(localStorage.getItem('known')) || []);
+  const [unknown, setUnknown] = useState(JSON.parse(localStorage.getItem('unknown')) || []);
+  const [all, setAll] = useState();
+
+
+  useEffect(() => {
+    shuffle(Questions);
+    checkAllWithUserBase(shuffledQuestions);
+  }, []);
+
+  useEffect(() => {
+    localStorage.setItem('known', JSON.stringify(known));
+    localStorage.setItem('unknown', JSON.stringify(unknown));
+  });
+
+  const whatsDisplayed = (dataFromIntro) => {
+    setShowIntro(dataFromIntro);
+  }
+
+  const checkAllWithUserBase = (questions) =>{
+    let check = questions.map(question => question.id);
+      for (let i = 0; i < known.length; i++) {
+          check = check.filter(id => id !== known[i]);
```

Comment:
try not to use "for", instead use just filter, rename check variable and give it to setAll (I hope it'll work)